### PR TITLE
Added topbar and headline sections and stylized

### PR DESCRIPTION
**Pull request by aghtetnyein**
Finished complete html and css styles for the first two sections which are topbar and headline.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+    />
+    <link rel="stylesheet" href="index.css" />
+    <title>SA AUNG HTET NYEIN</title>
+  </head>
+  <body>
+    <!-- the toolbar -->
+    <nav>
+      <p class="logo">SASA</p>
+      <div class="menu"><i class="fa-solid fa-bars"></i></div>
+    </nav>
```
Comment (Ugboaja-Uchechi) [inline index.html]: - [x] Great job using a `nav` tag to wrap your `menu` 👍  Kindly wrap your `nav` tag in a `header` tag. Your site's title or logo (or both) and the main site navigation should be wrapped in a `header` tag. 🙏

File: index.html [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.2/css/all.min.css"
+    />
+    <link rel="stylesheet" href="index.css" />
+    <title>SA AUNG HTET NYEIN</title>
+  </head>
+  <body>
+    <!-- the toolbar -->
+    <nav>
+      <p class="logo">SASA</p>
```
Comment (Ugboaja-Uchechi) [inline index.html]: - [x] Please make sure you wrap your logo in an `a` tag. 🙏  This is a good convention, semantically better, and would give a better user experience(UX)

File: index.html [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+        </a>
+        <a href="#">
+          <i class="fa-brands fa-medium"></i>
+        </a>
+      </div>
+      <img
+        class="illustration"
+        src="./assets/images/headline-section-illustration.png"
+        alt=""
+      />
```
Comment (Ugboaja-Uchechi) [inline index.html]: - [x] Kindly make sure this is a `background-image` not an `img` tag. 🙏  If an image isn't part of your website content but for aesthetic reasons then a `background-image` is preferred. You can read more about it [here](https://stackoverflow.com/questions/492809/when-to-use-img-vs-css-background-image)

File: index.html [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+      <p class="about-me">
+        A software engineering student with holistic knowledge of software
+        development with over 4 years of web development experience. Passionate
+        about UI/UX design and the art of photography. Extremely motivated to
+        develop my skills and grow professionally.
+      </p>
+      <div class="social-accounts">
+        <a href="#">
+          <i class="fa-brands fa-github"></i>
+        </a>
```
Comment (Ugboaja-Uchechi) [inline index.html]: - [x] Great job wrapping your icons in an `a` tag. 👏  Kindly wrap the icons in a `ul` tag and each of the icons in an `li` tag 🙏

File: README.md [hunk 5]
```diff
@@ -1,2 +1,17 @@
...
+
+- HTML, CSS, Javascript
+
+👤 **Sa Aung Htet Nyein (SASA)**
+
+- GitHub: [@aghtetnyein](https://github.com/aghtetnyein)
+
+## 📝 License
+
+This project is [MIT](./LICENSE) licensed.
```
Comment (Ugboaja-Uchechi) [inline README.md]: - [ ] Kindly make sure your `Readme` file has all the necessary sections as seen in the [Microverse Readme template](https://github.com/microverseinc/readme-template/blob/master/README.md) provided 🙏

**Review by Ugboaja-Uchechi — CHANGES_REQUESTED** [review]
Hi @aghtetnyein, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

- Linters are passing ✔️ 
- Pull request is descriptive 👏 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Ismailco — APPROVED** [review]
### Hi @aghtetnyein, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

cr-approve


## Highlights

✔️  No linter errors
✔️ Used Github Flow
✔️ Work documented professionally

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**